Plugins/Tweaks/Tweaks/FixUnlimitedPotionsBug.cpp

```diff
@@ -0,0 +1,36 @@
+#include "Tweaks/FixUnlimitedPotionsBug.hpp"
+
+#include "Services/Hooks/Hooks.hpp"
+#include "Utils.hpp"
+
+#include "API/Functions.hpp"
+
+namespace Tweaks {
+
+using namespace NWNXLib;
+using namespace NWNXLib::API;
+
+bool s_bUsableItemRemoval = false;
```

Comment:
If it is `s_`, it should be `static`.

Reply:
Done. Is a static global OK or should I make it a static member instead?